student-work/felice-berk/API_Final_Project/POST.md

```diff
@@ -58,8 +58,8 @@ Beneath the code snippet is a table explaining the components of the POST.
 | pattyWeightG  	| int       	| 160, 220, 330                         	| This indicates the weight of the patty in grams. The limit is 330 grams.             	|
 | pattyCook     	| string    	| "MR", "M", "MW" , "WD"                | The doneness of the patty. MR-medium rare, M-medium, MW-medium well, WD-well done.                    	|
 | bunType       	| string    	| "white", "wholeWheat", "glutenFree"               	| Choice of bun types including white, whole wheat, and gluten free.        	|
-| condiment(x) 	| string    	| "ketchup", "barbecueSauce", "spicyMayo", "none"  	| Choice of up to three condiments. A choice of none means no condiments.              	|
-| topping(x)   	| string    	| "lettuce", "tomato", "onion", "pickles", "none" 	| Choice of up to four toppings. A choice of none means no topping.                 	|
+| condiment``x`` 	| string    	| "ketchup", "barbecueSauce", "spicyMayo", "none"  	| Choice of up to three condiments. A choice of none means no condiments.              	|
```

Comment:
```suggestion
| condiment``x`` 	| string    	| "ketchup", "barbecueSauce", "spicyMayo", "none"  	| Choice of up to three condiments. A choice of none means no condiments.              	|
```

Reply:
Revert this back to the (x) but change the "quotes" for "ketchup" and the like to be done in the way I changed line 59.